docs: [MLT-62] Explicit about [vision] option (#153)
encord-team/encord-agents: docs/installation.md

python -m pip install encord-agents
```

> Note that we also optionally include additional dependencies used for working with Visual files: e.g: Images, Videos, Image Groups / Sequences. These can be used by

Reviewer: ```suggestion
You can optionally include additional dependencies used for working with visual files such as Images, Videos, Image Groups / Sequences by running:
```

```shell
python -m pip install encord-agents[vision]
```

!!! info

>     We recommend this for the majority of use cases as we explicitly build our methods to make working with Visual files as easy and as fast as possible. We offer it as an optional dependency to support serverless use cases where you wish to avoid bundle size or are making Agents that don't interact with the visual data in each task e.g: Working purely with label data / priorities, text / pdf files or Audio files.

Reviewer: ```suggestion
    We recommend installing this for most use cases, as our methods are optimized for fast, easy handling of visual files. It is optional to support serverless setups where you wish to avoid bundle size, or Agents that do not need visual data — such as those working only with labels, priorities, text, PDFs, or audio. 
```

## Dependencies

> The dependencies for `encord-agents` are kept lightweight, with the only relatively large dependencies being `numpy` with `opencv-python-headless` included in `encord-agents[vision]` also. To see the full list of dependencies, go [here](https://github.com/encord-team/encord-agents/blob/main/pyproject.toml){ target="\_blank", rel="noopener noreferrer" }.

Reviewer: ```suggestion
The dependencies for `encord-agents` are kept lightweight, with the largest dependencies being `numpy` with `opencv-python-headless` also included in `encord-agents[vision]`. To see the full list of dependencies, go [here](https://github.com/encord-team/encord-agents/blob/main/pyproject.toml){ target="\_blank", rel="noopener noreferrer" }.
```
